.github/workflows/liam-erd-sample-e2e_tests.yml

```
@@ -0,0 +1,135 @@
+          puts "Should Run: #{result}"
+          puts "Concurrency Group: liam-erd-sample-e2e-tests-#{target_url}"
+
+          # Same as 'echo "should_run=xxxx" >> $GITHUB_OUTPUT'
+          output_file = ENV.fetch("GITHUB_OUTPUT")
+          File.open(output_file, "a") do |file|
+            file.puts result
+          end
+
+      - name: Setup pnpm
+        if: steps.check.outputs.should_run == 'true'
+        uses: ./.github/actions/pnpm-setup
```

actionlint (1.7.7)

description is required in metadata of "pnpm-setup" action at "/home/jailuser/git/.github/actions/pnpm-setup/action.yml"

(action)